lib/internal/test_runner/utils.js

```diff
@@ -74,10 +87,72 @@ function convertStringToRegExp(str, name) {
   }
 }
 
+const kBuiltinDestinations = new SafeMap([
+  ['stdout', process.stdout],
+  ['stderr', process.stderr],
+]);
+
+const kBuiltinReporters = new SafeMap([
+  ['spec', 'node:test/reporter/spec'],
+  ['dot', 'node:test/reporter/dot'],
+  ['tap', 'node:test/reporter/tap'],
+]);
+
+const kDefaultReporter = 'tap';
+const kDefaltDestination = 'stdout';
+
+async function getReportersMap(reporters, destinations) {
+  const result = await SafePromiseAll(ArrayPrototypeMap(reporters, async (name, i) => {
+    const destination = kBuiltinDestinations.get(destinations[i]) ?? createWriteStream(destinations[i]);
+    let reporter = await requireOrImport(kBuiltinReporters.get(name) ?? name);
+
+    if (reporter && reporter.default) {
+      reporter = reporter.default;
+    }
+
+    if (reporter.prototype && ObjectGetOwnPropertyDescriptor(reporter.prototype, 'constructor')) {
+      reporter = new reporter();
+    }
+
+    if (!reporter) {
+      throw new ERR_INVALID_ARG_VALUE('Reporter', name, 'is not a valid reporter');
+    }
+
+    return { __proto__: null, reporter, destination };
+  }
+  ));
```

Comment:
```suggestion
  }));
```
so it matches the opening line - this confused me as is